shared/wallets/wallet/settings/popups/really-remove-account/index.js

```diff
@@ -46,7 +47,7 @@ class ReallyRemoveAccountPopup extends React.Component<Props, State> {
             fullWidth={Styles.isMobile}
             key={1}
             label="Finish"
-            onClick={this.props.onClose}
+            onClick={this.props.onFinish}
```

Comment:
The design wasn't super clear, but I think it makes sense to call the RPC to delete an account after a user clicks the "Finish" button on the `ReallyRemovePopup`. Thus we need to distinguish between a successful finish and a cancel with an extra prop